uptane-standard.md

```diff
@@ -599,29 +567,33 @@ For example:
 
 An Uptane implementation SHALL make the following services available to vehicles:
 
-* Image repository
-* Director repository
+* A Director repository
+* At least one Image repository
+
+Additionally, an Uptane implementation requires ECUs to have a secure way to know the current time; this MAY be implemented as a server.
 
-Additionally, an Uptane implementation requires ECUs to have a secure way to know the current time. This SHOULD be accomplished using a time server ({{time_server}}), but MAY be implemented in other ways.
+Uptane implementations MAY contain multiple repositories. The simplest case is to have a Director repository and a single Image repository. More complex configurations are possible, however.
```

Comment:
The right way to reference the deployment considerations document is with an informative reference that can then be cited. This is one of several places where it would be useful to cite the deployment considerations, but we don't have anything to cite yet.